library/alloc/src/collections/btree/map.rs

```diff
@@ -815,7 +811,7 @@ impl<K: Ord, V> BTreeMap<K, V> {
     /// types that can be `==` without being identical. See the [module-level
     /// documentation] for more.
     ///
-    /// [module-level documentation]: index.html#insert-and-complex-keys
+    /// [module-level documentation]: crate#insert-and-complex-keys
```

Comment:
```suggestion
    /// [module-level documentation]: self#insert-and-complex-keys
```

Reply:
That should be it, but I get the following error. I think we had that same problem with another link to a module level doc.

```
814 |     /// [module-level documentation]: self#insert-and-complex-keys
    |                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ this item is private
```

Reply:
Hmm, maybe this is meant to be shown on the re-export? Try `super#` and see if that helps/generates the right link.

Reply:
I tried your suggestion, but it didn't help. I got the same error.
```
814 |     /// [module-level documentation]: super#insert-and-complex-keys
    |                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ this item is private
```

Reply:
Ok, I took the time to look it up - it's linking to https://doc.rust-lang.org/std/collections/index.html#insert-and-complex-keys. Wow, that is not clear at all from the current text.

Try `[module-level documentation]: crate::collections#insert-and-complex-keys`.

Reply:
This looks good, thanks for helping me out.